test/test_custom_ops.py

```diff
@@ -865,6 +866,24 @@ def foo(x: Tensor, y: Tuple[int, int]) -> Tensor:
 
             del foo
 
+        with self.assertRaisesRegex(ValueError, r"For example, typing.List\[int\]"):
+            # test that we propose a correct and supported type.
+            @custom_ops.custom_op(f"{TestCustomOp.test_ns}::foo")
+            def foo(x: Tensor, y: Tuple[int, int]) -> Tensor:
+                raise NotImplementedError
+
+            del foo
+
+        with self.assertRaises(ValueError) as cm:
+
+            @custom_ops.custom_op(f"{TestCustomOp.test_ns}::foo")
```

Comment:
This might need to be `torch.library.custom_op`, I think `custom_ops.custom_op` is the older custom ops prototype that I didn't delete

Reply:
fixed. I see that some other test cases are using `custom_ops.custom_op`, do we want to change them all to `torch.library.custom_op`?